doc/stdlib/base.rst

```diff
@@ -654,143 +891,167 @@ System
+
+   **Examples**:     * "run(pipe(``ls``, ``grep xyz``))"
+
+   ::
+
+       * "run(pipe(`ls`, "out.txt"))"
+
+       * "run(pipe("out.txt", `grep xyz`))"
+
+       pipe(command; stdin, stdout, stderr, append=false)
+
+   Redirect I/O to or from the given "command". Keyword arguments specify which of the command's streams should be redirected. "append" controls whether file output appends to the file. This is a more general version of the 2-argument "pipe" function. "pipe(from, to)" is equivalent to "pipe(from, stdout=to)" when "from" is a command, and to "pipe(to, stdin=from)" when "from" is another kind of data source.
+
+   **Examples**:     * "run(pipe(``dothings``, stdout="out.txt",       stderr="errs.txt"))"
+
+   ::
+
+       * "run(pipe(`update`, stdout="log.txt", append=true))"
+
 
 .. function:: pipe(from, to, ...)
 
-   Create a pipeline from a data source to a destination. The source and destination can
-   be commands, I/O streams, strings, or results of other ``pipe`` calls. At least one
-   argument must be a command. Strings refer to filenames.
-   When called with more than two arguments, they are chained together from left to right.
-   For example ``pipe(a,b,c)`` is equivalent to ``pipe(pipe(a,b),c)``. This provides a more
-   concise way to specify multi-stage pipelines.
+   Create a pipeline from a data source to a destination. The source and destination can be commands, I/O streams, strings, or results of other "pipe" calls. At least one argument must be a command. Strings refer to filenames. When called with more than two arguments, they are chained together from left to right. For example "pipe(a,b,c)" is equivalent to "pipe(pipe(a,b),c)". This provides a more concise way to specify multi-stage pipelines.
+
+   **Examples**:     * "run(pipe(``ls``, ``grep xyz``))"
+
+   ::
+
+       * "run(pipe(`ls`, "out.txt"))"
+
+       * "run(pipe("out.txt", `grep xyz`))"
+
+       pipe(command; stdin, stdout, stderr, append=false)
 
-   **Examples**:
-     * ``run(pipe(`ls`, `grep xyz`))``
-     * ``run(pipe(`ls`, "out.txt"))``
-     * ``run(pipe("out.txt", `grep xyz`))``
+   Redirect I/O to or from the given "command". Keyword arguments specify which of the command's streams should be redirected. "append" controls whether file output appends to the file. This is a more general version of the 2-argument "pipe" function. "pipe(from, to)" is equivalent to "pipe(from, stdout=to)" when "from" is a command, and to "pipe(to, stdin=from)" when "from" is another kind of data source.
 
-.. function:: pipe(command; stdin, stdout, stderr, append=false)
+   **Examples**:     * "run(pipe(``dothings``, stdout="out.txt",       stderr="errs.txt"))"
```

Comment:
Note: This particular one should be ``run(pipe(`dothings`, stdout=...`` (it's grabbed the wrong line to diff against).